Makefile

```diff
@@ -77,7 +77,7 @@ docker: ## Builds docker image
 generate-client-code: ## generate API client code
 	docker run --rm \
 		-v ${PWD}:/local openapitools/openapi-generator-cli:${OPENAPI_GENERATOR_VERSION} generate \
-		-i /local/api/api_manager.swagger.yaml \
+		-i https://docs.api.stackit.cloud/oas/api-manager \
```

Comment:
This cannot work. 
The API Manager is not meant to be public. I will bring the topic up in the daily.